tests/test_perf_error_already_set.py

```diff
@@ -0,0 +1,58 @@
+import time
+
+import pytest
+
+from pybind11_tests import perf_error_already_set as m
+
+
+def find_call_repetitions(
```

Comment:
Why don't we just go ahead and just use pytest-benchmark already? https://pypi.org/project/pytest-benchmark/